extension/src/test/suite/Formatter_mock.test.ts

```diff
@@ -0,0 +1,392 @@
+    }
+  });
+
+  test("Lisp Formatter Test case 13", async function () {
+    // Test Mixed settings
+    // MaxLineChars: 65
+    // NarrowStyleIndent: 4
+    // CloseParenthesisStyle: 'Same line'
+    // LongListFormatStyle: 'Single line'
+    let i = 13;
+    try {
+      const [source, output, baseline] = getFileName(i);
+      let doc = ReadonlyDocument.open(source);
+      await setClosedParenInSameLine("same line");
+      await setIndentSpaces(4);
+      await setMaxLineChars(65);
+      await setLongListFormat("single column");
+      let fmt = LispFormatter.format(doc, null);
+      comparefileSync(i, output, fmt, baseline);
+    } catch (err) {
+      assert.fail(`The lisp format test case ${i} failed`);
+    }
+  });
+
+  test("Lisp Formatter Test case 14", async function () {
+    // Test the comments after the brackets ") ;progn"
+    // MaxLineChars: 80
+    // NarrowStyleIndent: 4
+    let i = 14;
+    try {
+      const [source, output, baseline] = getFileName(i);
+      let doc = ReadonlyDocument.open(source);
+      await setIndentSpaces(4);
+      await setMaxLineChars(80);
+      let fmt = LispFormatter.format(doc, null);
+      comparefileSync(i, output, fmt, baseline);
+    } catch (err) {
+      assert.fail(`The lisp format test case ${i} failed`);
+    }
+  });
+
+  test("Lisp Formatter Test case 15", async function () {
+    // Test unicode
+    // MaxLineChars: 60
+    // NarrowStyleIndent: 2
+    let i = 15;
+    try {
+      const [source, output, baseline] = getFileName(i);
+      let doc = ReadonlyDocument.open(source);
+      await setIndentSpaces(2);
+      await setMaxLineChars(60);
+      let fmt = LispFormatter.format(doc, null);
+      comparefileSync(i, output, fmt, baseline);
+    } catch (err) {
+      assert.fail(`The lisp format test case ${i} failed`);
+    }
+  });
+
+  test("Lisp Formatter Test case 16", async function () {
+    // Test invalid setting
+    // MaxLineChars: 30
+    // NarrowStyleIndent: 8
+    let i = 16;
+    try {
+      const [source, output, baseline] = getFileName(i);
+      let doc = ReadonlyDocument.open(source);
+      await setIndentSpaces(8);
+      await setMaxLineChars(30);
+      let fmt = LispFormatter.format(doc, null);
+      comparefileSync(i, output, fmt, baseline);
+    } catch (err) {
+      assert.fail(`The lisp format test case ${i} failed`);
+    }
+  });
+
+  async function setIndentSpaces(indent: number) {
+    if (indentSpacesStub) {
+      indentSpacesStub.restore();
+    }
+    indentSpacesStub = ImportMock.mockFunction(
+      fmtConfig,
+      "indentSpaces",
+      indent
+    );
+  }
+  async function setClosedParenInSameLine(closeParenStyle: string) {
+    if (closeParenStyleStub) {
+      closeParenStyleStub.restore();
+    }
+    closeParenStyleStub = ImportMock.mockFunction(
+      fmtConfig,
+      "closeParenStyle",
+      closeParenStyle
+    );
+  }
+  async function setMaxLineChars(maxchar: number) {
```

Comment:
there's no 'await' in this method, so I there's no need to mark it as async or to use await to call this method

Reply:
I second that.

Reply:
Will update.